builtin/providers/aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -149,6 +204,76 @@ resource "aws_elasticache_cluster" "bar" {
     port = 11211
     parameter_group_name = "default.memcached1.4"
     security_group_names = ["${aws_elasticache_security_group.bar.name}"]
+    snapshot_window = "05:00-09:00"
+    snapshot_retention_limit = 3
```

Comment:
Engine `memcached` does not support snapshotting, so the `TestAccAWSElasticacheCluster_basic` test now fails with this addition :/